src/tools/dev/scripts/bv_support/bv_adios.sh

```diff
@@ -258,14 +258,14 @@ function apply_adios_patch
 function build_adios
 {
     #
-    # Prepare build dir
+    # Uncompress the source file
     #
-    prepare_build_dir $ADIOS_BUILD_DIR $ADIOS_FILE
+    uncompress_src_file $ADIOS_SRC_DIR $ADIOS_FILE
     untarred_ADIOS=$?
     # 0, already exists, 1 untarred src, 2 error
 
     if [[ $untarred_ADIOS == -1 ]] ; then
-        warn "Unable to prepare ADIOS Build Directory. Giving Up"
+        warn "Unable to uncompress ADIOS Build Directory. Giving Up"
```

Comment:
If we look at what is really changing either in your edits here or in the contents of these warning messages from module to module, its simple words like `uncompress` vs `prepare` or `ADIOS` vs `HDF5`. I don't see the need to have whole *functions* in `bv_xxx.sh` files to support these minor variations of behavior.

Also, I am not sure it makes sense to *uncompress* a *directory*.

Reply:
I tried to do some scripted changes, but did not get all of them so the message make sense. That is one of them. I agree the about the need to have whole functions in bv_xxx.sh files to support these minor variations of behavior. 

Sidebar: When I tested conduit with the new construct_build_visit_module.sh  the following needed customization: bv_conduit_depends_on, bv_conduit_info, bv_conduit_host_profile, apply_conduit_patch, build_conduit (only the configure part). Everything else was verbatim. So yes lots of unneeded stuff. That could all part of a redesign.